includes/wikia/parser/templatetypes/handlers/NavboxTemplate.class.php

```diff
@@ -28,24 +28,39 @@ private static function process( $html ) {
 		preg_match_all( $markerRegex, $html, $markers );
```

Comment:
would it make sense to move this `$markerRegex` to const or a separate function to not to have to C&P it in tests?https://github.com/Wikia/app/pull/10466/files#diff-ed50d154df18b779157b998300265e66R28